index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+    <script src="https://kit.fontawesome.com/b2291adafa.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>Mobile version</title>
+</head>
+<body>
+        <section class="main-container">
+           <Section class="header">
+               <Nav>
+                   <a href=""><h3>My Logo</h3></a>
+                   <span class="material-icons">menu</span>
+               </Nav>
+           </Section>
+           <section class="headline">
```

Comment:
- [ ] You are almost there! However, as the previous reviewer said, it would be better to use a `header` tag for this part.